cloud/src/meta-service/meta_service_txn.cpp

```diff
@@ -1259,6 +1259,22 @@ void commit_txn_immediately(
     // Finally we are done...
     err = txn->commit();
     if (err != TxnErrorCode::TXN_OK) {
+        if (err == TxnErrorCode::TXN_VALUE_TOO_LARGE || err == TxnErrorCode::TXN_BYTES_TOO_LARGE) {
+            DCHECK(!tmp_rowsets_meta.empty());
```

Comment:
it's better to return log warning directly instead of DCHECK and continue to dereferencing tmp_rowset_meta